core/src/main/groovy/com/muwire/core/files/PersisterService.groovy

```diff
@@ -46,15 +33,12 @@ class PersisterService extends Service {
 
     void stop() {
         timer.cancel()
+        persisterExecutor.shutdown()
```

Comment:
I don't see a `persisterExecutor` in scope here, am I missing something?

Reply:
It's here https://github.com/zlatinb/muwire/pull/37/files/855183397bd8bcd92451dbaf4d7e1a8f3ed6bed4#diff-5a366551994971557c2386775e63a9bdR22 . A field of the class

Reply:
But that's line 35 in `PersisterFolderService`, not `BasePersisterService`, no?

Reply:
I'm confused. This comment is in `PersisterFolderService` which has `PersisterFolderService::persisterExecutor`. `BasePersisterService` doesn't have an executor. Do you want me to move the persisterExecutor up to `BasePersisterService` or remove it entirely from `PersisterFolderService`?

Reply:
Huh?? This comment should be for `PersisterService` is GitHub playing tricks?

Reply:
What in the world? Well now your comment makes more sense. I'll get rid of the executor in `PersisterService`.